breakingChangePost.md

```diff
@@ -0,0 +1,58 @@
+An upcoming release of the **AWS SDK for Kotlin** will introduce an optional region provider parameter that determines the AWS region when configuring a client
+
+# Release date
+
+This change will be included in the upcoming **v1.5.0** release,
```

Comment:
nit: trailing comma. Typically we include an estimated release date but it's a few months out right now.